posthog/constants.py

```diff
@@ -107,6 +107,8 @@
 PATHS_INCLUDE_EVENT_TYPES = "include_event_types"
 PATHS_INCLUDE_CUSTOM_EVENTS = "include_custom_events"
 PATHS_EXCLUDE_EVENTS = "exclude_events"
+FUNNEL_PATH_JOURNEY = "funnel_path_journey"
```

Comment:
I found these names a bit confusing.

We're looking at 3 cases:
1. Where did Users go after Step X (this is I think `FUNNEL_PATH_DROPOFF` )
2. What did Users do between steps ( `FUNNEL_PATH_JOURNEY` )
3.  What else did Users do before Step X (this is like the reverse of `FUNNEL_PATH_DROPOFF` )

Thoughts on calling them
1. `FUNNEL_PATH_AFTER_STEP`
2. `FUNNEL_PATH_BETWEEN_STEPS`
3. `FUNNEL_PATH_BEFORE_STEP`

Reply:
yeah this is better